Makefile

```diff
@@ -1,51 +1,26 @@
-
-# Default values if not already set
-PGOROOT ?= $(CURDIR)
-PGO_BASEOS ?= ubi8
-PGO_IMAGE_PREFIX ?= crunchydata
-PGO_IMAGE_TAG ?= $(PGO_BASEOS)-$(PGO_VERSION)
-PGO_VERSION ?= $(shell git describe --tags)
-PGO_PG_VERSION ?= 14
-PGO_PG_FULLVERSION ?= 14.6
-PGO_KUBE_CLIENT ?= kubectl
```

Comment:
Should a default be kept for `PGO_KUBE_CLIENT`?

Reply:
Looks like we'll need to update `check-envtest-existing` and `check-kuttl` to use `kubectl`.